packages/starlight/components/Banner.astro

```diff
@@ -0,0 +1,22 @@
+---
+interface Props {
+  content: string;
+}
+---
+
+<div class="sl-banner" role="banner" set:html={Astro.props.content} />
+
+<style>
+  .sl-banner {
+    --__sl-banner-text: var(--sl-color-banner-text, var(--sl-color-text-invert));
+    padding: var(--sl-nav-pad-y) var(--sl-nav-pad-x);
+    background-color: var(--sl-color-banner-bg, var(--sl-color-bg-accent));
+    color: var(--__sl-banner-text);
+    line-height: var(--sl-line-height-headings);
+    text-align: center;
```

Comment:
```suggestion
    text-align: center;
    text-wrap: balance;
```
If progressive CSS enhancements are something Starlight is open to then you could add this one: https://caniuse.com/css-text-wrap-balance, https://www.w3.org/TR/css-text-4/#text-wrap

Reply:
Definitely! And I think this is safe. Even if people can theoretically provide all kinds of content, I don't believe this would disrupt that.

Reply:
Definitely! And I think this is safe. Even if people can theoretically provide all kinds of content, I don't believe this would disrupt that.